source/_integrations/tesla_fleet.markdown

```
@@ -40,30 +40,124 @@
+   - If unsure, you can click '**Skip & Submit**' for now and add the billing details later when your usage is close to the free threshold.
+9. Your developer application is ready for use!
+
+{% enddetails %}
+
+{% details "Register your application as a Fleet API partner" %}
+
+These steps are also summarized in the [Tesla Fleet API documentation Step 4](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-4-call-the-register-endpoint), but the steps below provide easier copy-pasteable code and additional checks.
+
+1. Get your OAuth details by going to your [Developer dashboard](https://developer.tesla.com/en_US/dashboard), clicking '**View Details**' under the app you set up for Home Assistant integration, then clicking on the '**Credentials & APIs**' tab. Note the `Client ID` and `Client Secret` strings.
+2. Set the following variables and run this CURL request:
+   ```
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)